docs/development/testing.md

```diff
@@ -163,6 +164,23 @@ When compiled, lc-compile will emit an error on the 'put' line because tInnerVar
 
 To help debug compiler tests, set the LCC_VERBOSE environment variable to 1 before running the compiler test. This will cause the compiler testrunner to emit diagnostic information, including the full output of the compile command which is being run.
 
+### LiveCode Script Parser Tests
+
+The syntax for LiveCode Script parser tests is the same as that of the 
+LCB compiler frontent tests above. LCS parser test files all have the 
+extension '.parsertest'.
+
+Expected errors are referred to by their error code, for example the 
+following tests the variable shadowing parse error with code 220, 
+"local: name shadows another variable or constant":
+
+	%TEST ShadowVariable
+	local sVar
+	local %{BEFORE_SHADOW}sVar
+	%EXPECT PASS
+	%ERROR 220 AT BEFORE_SHADOW
```

Comment:
I'm wondering whether it is wise to rely on numeric errors here - could we parse the errors file in the engine/src tree so we can use names?